network-api/networkapi/campaign/views.py

```diff
@@ -125,6 +125,7 @@ def petition_submission(request, petition):
         "email": request.data['email'],
         "email_subscription": request.data['newsletterSignup'],
         "source_url": request.data['source'],
+        "page_locale": request.data['locale'],
```

Comment:
Is this the proper way to send locale data to backend? 😬 @Pomax

Reply:
According to https://basket.readthedocs.io/newsletter_api.html?highlight=country#news-subscribe this should be `lang`, not `page_locale`

Reply:
It doesn't look like basket is expecting a language at all for now 

https://github.com/mozmeao/basket/blob/23c8b7ce3ded8b82107b985d78d8827f2e9060b8/basket/news/tasks.py#L1057-L1112

Reply:
You can see on line 1086 that we hard-code `en-US` if they've asked for a newsletter subscription

Reply:
so `lang` would be fine, but I can add support for `page_locale` just as easily

Reply:
Just file a basket issue for that when you're ready with all the necessary changes.

Reply:
thanks!